pkg/cdn/metadata/rdbms/rdbms.go

```diff
@@ -0,0 +1,31 @@
+package rdbms
+
+import (
+	"github.com/gobuffalo/pop"
+)
+
+// MetadataStore represents a rdbms(postgres, mysql, sqlite, cockroachdb) backed metadata store.
+type MetadataStore struct {
+	conn           *pop.Connection
+	connectionName string // settings name from database.yml
+}
+
+// NewRDBMSStorage  returns an unconnected RDBMS Metadata Storage
+// that satisfies the Getter and Setter interfaces. You must call
+// Connect() on the returned store before using it.
+// connectionName
+func NewRDBMSStorage(connectionName string) *MetadataStore {
```

Comment:
This name also "stutters" because it will be called with `rdbms.NewRDBMSStorage` - can you change this name to `NewStorage` instead?